extractor/types.go

```diff
@@ -11,6 +12,13 @@ type Metadata struct {
 	Deposit string `json:"deposit"` // the deposit associated with the deployment
 }
 
+// TxData contains the single block transaction,
+// along with the block information
+type TxData struct {
+	Tx       std.Tx `json:"tx"`
+	BlockNum uint64 `json:"blockNum"`
+}
+
```

Comment:
why did you copy the data struct instead of using the exported one on tx-archive?

Reply:
Imported tx-archive types: [391d2ec](https://github.com/gnolang/tx-exports/pull/10/commits/391d2ecf46f86a829d1ddc2ce6792f20c03bad32)